portfolio/src/main/webapp/style.css

```diff
@@ -24,4 +24,9 @@ footer {
   padding: 3px;
   background-color: DarkSalmon;
   color: white;
-}
\ No newline at end of file
+}
+
+#padding {
```

Comment:
padding should be a CSS class, since CSS IDs are unique identifiers (only one per page).

Reply:
solved